src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -291,11 +301,7 @@
 @end
 
 @private processResponse(sample)
-  @if sample.outputs.size == 0
-    System.out.println(response);
-  @else
-    {@processOutputViews(sample.outputs)}
-  @end
+  {@processOutputViews(sample.outputs)}
```

Comment:
For concision, consider always calling a variant of this, eg `printResponse` from up above and having the `if sample.outputs` inside it. That will get rid of a lot of repeated code that, right now, the caller is responsible for. (I realize you still need `processResponse` as is for exceptional cases like the one in l416.

Similarly for the other languages, where most calls are currently wrapped in an `if`. Make the caller have less responsibility ;-)

Reply:
Yes that's definitely a better approach. However, if we put the empty check inside a wrapper function, it will result in an empty line in the generated code if the `sample.outputs` is empty (idk why the `snip` files are evaluated like that, which totally does not make sense to me). Sometimes it's OK, but sometimes it's really annoying (like, a blank line between two closed curly braces :()